src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -353,42 +356,31 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
                 }
             }
             case ChartTypeEnum.SCATTER: {
-                // sampleToAnalysisGroup is complete because of loadingPromises and StudyViewComponentLoader
                 return (
-                    <StudyViewScatterPlot
-                        ref={this.handlers.ref}
-                        width={400}
-                        height={380}
-                        onSelection={this.props.onValueSelection}
-                        data={this.props.promise.result}
-                        isLoading={this.props.selectedSamples.isPending}
-
-                        sampleToAnalysisGroup={this.props.sampleToAnalysisGroup!.result!}
-                        analysisGroups={this.props.analysisGroupsSettings.groups}
-                        analysisClinicalAttribute={this.props.analysisGroupsSettings.clinicalAttribute}
+                    <div style={{marginTop:-33}}>
```

Comment:
What's this for?

Reply:
this is to put less space between title and chart, I could not make the adjustment in victory so I just had to make the SVG bigger and then move it up